ios/Capacitor/Capacitor/CAPBridgeViewController.swift

```diff
@@ -105,7 +105,12 @@ import Cordova
         webViewConfiguration.allowsAirPlayForMediaPlayback = true
         webViewConfiguration.mediaTypesRequiringUserActionForPlayback = []
         if let appendUserAgent = instanceConfiguration.appendedUserAgentString {
-            webViewConfiguration.applicationNameForUserAgent = appendUserAgent
+            guard webViewConfiguration.applicationNameForUserAgent != nil else {
+                webViewConfiguration.applicationNameForUserAgent = appendUserAgent
+                return webViewConfiguration;
+            };
+            
+            webViewConfiguration.applicationNameForUserAgent! += " " + appendUserAgent;
```

Comment:
While it is true that force unwrapping `applicationNameForUserAgent` should be safe here because it has already been checked for nil, it will still generate a warning from the linter because it doesn't know that. And the string interpolation could be easier to mentally parse.
```suggestion
            if let appName = webViewConfiguration.applicationNameForUserAgent {
                webViewConfiguration.applicationNameForUserAgent = "\(appName)  \(appendUserAgent)"
            } else {
                webViewConfiguration.applicationNameForUserAgent = appendUserAgent
            }
```